provider/pkg/gen/schema.go

```diff
@@ -91,6 +101,10 @@ func PulumiSchema(swagger map[string]interface{}) pschema.PackageSpec {
 					Description: "BETA FEATURE - If present and set to true, enable server-side diff calculations.\nThis feature is in developer preview, and is disabled by default.\n\nThis config can be specified in the following ways, using this precedence:\n1. This `enableDryRun` parameter.\n2. The `PULUMI_K8S_ENABLE_DRY_RUN` environment variable.",
 					TypeSpec:    pschema.TypeSpec{Type: "boolean"},
 				},
+				"renderYamlToDirectory": {
```

Comment:
The old C# SDK has this property - should we keep it?

Reply:
Yes, that was an oversight, and it should still be present. This should fix #1083